casadm/cas_lib.c

```diff
@@ -1915,6 +1915,29 @@ int core_pool_remove(const char *core_device)
 	return SUCCESS;
 }
 
+int purge_cache(unsigned int cache_id)
+{
+	int fd = 0;
+	struct kcas_flush_cache cmd;
+
+	fd = open_ctrl_device();
+	if (fd == -1)
+		return FAILURE;
+
+	memset(&cmd, 0, sizeof(cmd));
+	cmd.cache_id = cache_id;
+	/* synchronous flag */
+	if (run_ioctl_interruptible(fd, KCAS_IOCTL_PURGE_CACHE, &cmd, "Purging cache",
+			cache_id, OCF_CORE_ID_INVALID) < 0) {
+		close(fd);
```

Comment:
I think `print_err()` will provide enough information about reasons why purge failed, however if have idea for any particular interesting and not redundant message we can talk about it. I also kept in mind that purge is hidden command and it will be used for testing purposes, not in any production env